deploy/crds/infra.watch_v1beta1_servicetelemetry_cr.yaml

```diff
@@ -33,6 +33,8 @@ spec:
           strategy: persistent
           persistent:
             pvcStorageRequest: 20Gi
+      endpoint_cert_duration: 70080h
```

Comment:
The use of the snake_case is part of the conversion from camelCase to the Ansible workflow. The CRD should be in camelCase, not snake_case.